### DISCUSSION TITLE
metadata migration error 0.10.0 => 0.11.0

### DISCUSSION THREAD

**Discussion by tjmahr** [ropensci/targets#812] [General]
[I had a problem and couldn't reproduce the solution after posting.]

**Comment by wlandau** [comment]
Sorry about that TJ. The metadata did change because of https://github.com/ropensci/targets/pull/808, but I thought I did that back-compatibly. fee039f2c39a5bdd018d62742a4d665d7c2f9207 should fix the problem you originally posted.
